src/index.js

```diff
@@ -3,6 +3,13 @@ import chProcess from 'child_process';
 import _ from 'lodash';
 import fs from 'fs';
 import chokidar from 'chokidar';
+import path from 'path';
+
+(function setPath() {
+  process.env.PATH = [
+    `${process.cwd()}/node_modules/.bin/`, process.env.PATH 
+  ].join(path.delimiter);
+})();
 
```

Comment:
Actually, `child_process.exec` has an `env` option that you can set. I think that's the ticket.